aws/resource_aws_placement_group.go

```diff
@@ -190,6 +197,9 @@ func resourceAwsPlacementGroupDelete(d *schema.ResourceData, meta interface{}) e
 			}
 
 			pg := out.PlacementGroups[0]
+			if *pg.State == "available" {
+				log.Printf("[DEBUG] Accepted status when deleting EC2 Placement group: %q %v", d.Id(), *pg.State)
+			}
```

Comment:
We can remove this logging 👍 Otherwise the code here should prevent panics by replacing the raw dereferencing with `aws.StringValue(pg.State)` in case it is `nil`.

Reply:
Agreed and done